### ISSUE TITLE
Setting Account Visible is not saved

### ISSUE THREAD

**Issue by Dandy64** [moneymanagerex/moneymanagerex#5795]
MMEX version:
 - [ ] 1.6.0
 - [ ] 1.6.1
 - [x] 1.6.2
 - [ ] 1.6.3 Beta
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

The setting which accounts should be visible is not saved after restarting the program.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:
1. Created 5 Cash Accounts.
2. Two of them closed (Account Status = Closed) => The closed accounts look pale.
3. Right click on Cash Accounts -> Accounts Visible ->Open
4. The closed accounts disappeared and are not shown anymore.
5. Restarted the Money Manager Ex

Expected result:
The closed accounts are not shown.
Actual result:
The closed accounts are visible.
# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by tactilis** [comment]
Duplicate of a previous issue. Please see https://github.com/moneymanagerex/moneymanagerex/issues/2635

Intended behaviour. Right-clicking and setting visibility is a temporary change.

You need to set your desired visibility in Options. That will be saved.

![image](https://user-images.githubusercontent.com/4060092/229900225-2d903d8c-076c-4451-b021-377e52945443.png)

**Comment by Dandy64** [comment]
You are right. So it is not an obvious behavior for users.

**Comment by tactilis** [comment]
You wouldn't want the temporary view of only Closed accounts to be stored as an Option setting that is persisted across MMEX restarts.

![image](https://user-images.githubusercontent.com/4060092/230607764-af0fb299-e898-4b31-ab2f-7c71bbb88a1c.png)

Showing only Closed accounts when MMEX is started is not useful; particularly so if you don't have any Closed accounts because then the entire Bank Accounts section of the Navigation tree disappears.

![image](https://user-images.githubusercontent.com/4060092/230608112-c5f18ece-ed17-4d10-b918-251462a38b3f.png)
